src/Shared/ServerInfrastructure/StringUtilities.cs

```diff
@@ -725,34 +725,24 @@ private static void PopulateSpanWithHexSuffix(Span<char> buffer, (string? str, c
 
             if (Ssse3.IsSupported)
             {
-                // These must be explicity typed as ReadOnlySpan<byte>
-                // They then become a non-allocating mappings to the data section of the assembly.
-                // This uses C# compiler's ability to refer to static data directly. For more information see https://vcsjones.dev/2019/02/01/csharp-readonly-span-bytes-static
```

Comment:
Please add a similar comment explaining the new usage pattern so someone doesn't try to 'optimize' it again.